common/app/views/fragments/audio/containers/flagshipContainer.scala.html

```diff
@@ -57,9 +58,24 @@
                     @getLatestEpisode(allCards).map { card =>
                         <div class="fc-item--type-media">
                             <div class="fc-podcast-container__episode">
-                                <div class="fc-podcast-container__episode-image">
-                                    <img src="@FlagshipFrontContainer.AlbumArtUrl"/>
-                                </div>
+                                @if(true){
+                                    @card.displayElement match {
+                                        case Some(InlineImage(image)) => {
+                                            <div class="fc-podcast-container__episode-image-story">
+                                                @itemImage(
```

Comment:
Hmm, that should insert screen ratio alternatives if I'm not mistaken 👍

Reply:
Oh i see so we are already using the scaling tool? I think the closeness of the crop may effect things by requesting an image that is slightly too small

Reply:
Yeah, that template is using [this one](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/image.scala.html) which provides access to all sources. That should be enough.